ste/concurrencyTuner.go

```diff
@@ -70,6 +71,9 @@ type autoConcurrencyTuner struct {
 		value  int
 		reason string
 	}
+	// atomicCurrentConcurrency should never be written to, it is a read only reference to the job manager
+	atomicCurrentConcurrency *int32
```

Comment:
where do we read this?

Reply:
Let's address this as a cleanup item but I dont see this being written to or read anywhere so I dont think we need this.